vmdb/app/models/miq_request.rb

```diff
@@ -72,6 +73,12 @@ class MiqRequest < ActiveRecord::Base
   REQUEST_TYPES_BACKEND_ONLY = {:MiqProvisionRequestTemplate => {:template => "VM Provision Template"}}
   REQUEST_TYPES = MODEL_REQUEST_TYPES.values.each_with_object(REQUEST_TYPES_BACKEND_ONLY) { |i, h| i.each { |k, v| h[k] = v } }
 
+  # TODO: delegate ":to => :class" and remove klass method once Rails 4 is merged
+  delegate :request_types, :task_description, :to => :klass
```

Comment:
@tenderlove Is there a better way to do this with respect to the Rails 4 upgrade?